api/app/routers/tickets.py

```diff
@@ -0,0 +1,82 @@
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, HTTPException, Query
+from sqlalchemy.orm import Session
+
+from app import command, database, models, persistence, schemas
+from app.auth.account import get_current_user
+
+router = APIRouter(prefix="/tickets", tags=["tickets"])
+
+
+def ticket_to_response(ticket: models.Ticket):
+    dependency = ticket.dependency
+    service = dependency.service
+    pteam_id = service.pteam_id
+    service_id = service.service_id
+    return schemas.TicketResponse(
+        ticket_id=UUID(ticket.ticket_id),
+        vuln_id=UUID(ticket.threat.vuln_id),
+        dependency_id=UUID(ticket.dependency_id),
+        service_id=service_id,
+        pteam_id=pteam_id,
+        created_at=ticket.created_at,
+        ssvc_deployer_priority=ticket.ssvc_deployer_priority,
+        ticket_safety_impact=ticket.ticket_safety_impact,
+        ticket_safety_impact_change_reason=ticket.ticket_safety_impact_change_reason,
+        ticket_status=(schemas.TicketStatusResponse.model_validate(ticket.ticket_status)),
+    )
+
+
+@router.get("", response_model=schemas.TicketListResponse)
+def get_tickets(
+    assigned_to_me: bool = Query(False),
+    pteam_ids: list[UUID] | None = Query(None),
+    offset: int = Query(0, ge=0),
+    limit: int = Query(100, ge=1, le=1000),
+    current_user: models.Account = Depends(get_current_user),
+    db: Session = Depends(database.get_db),
+):
+    """
+    Get paginated tickets related to the pteams the current user belongs to.
+    """
+
+    user_pteam_ids = {UUID(str(role.pteam_id)) for role in current_user.pteam_roles}
+    if not pteam_ids:
```

Comment:
```suggestion
    if not pteam_ids:
        user_pteam_ids = {UUID(str(role.pteam_id)) for role in current_user.pteam_roles}
```
ifの中でしか使わないため、ifの中で定義すべき

Reply:
修正しました